SECP-service/src/main/java/com/visucius/secp/resources/AdminResource.java

```diff
@@ -0,0 +1,38 @@
+package com.visucius.secp.resources;
+
+import com.visucius.secp.Controllers.Admin.AdminController;
+import io.dropwizard.auth.Auth;
+import io.dropwizard.hibernate.UnitOfWork;
+
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@RolesAllowed("ADMIN")
+@Path("/admin")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class AdminResource {
+    private final AdminController adminController;
+
+    public AdminResource(AdminController adminController) {
+        this.adminController = adminController;
+    }
+
+    @POST
+    @Path("{id}")
+    @UnitOfWork
+    public Response registerAdmin(@Auth @PathParam("id") String id) {
```

Comment:
Nice

Reply:
Yes lol. now we can register admin. I created 2 users with admin privileges.